src/lib/base58_check/version_bytes.ml

```diff
@@ -4,7 +4,7 @@ type t = char
 
 (* each of the following values should be distinct *)
 
-let graphql : t = '\x17'
+let user_command : t = '\x17'
```

Comment:
we should def document this somewhere

Reply:
document the particular version bytes used?